tools/deployment/osquery.example.conf

```diff
@@ -36,6 +36,9 @@
     // and query results differentials. See also 'use_in_memory_database'.
     //"database_path": "/var/osquery/osquery.db",
 
+    // A filesystem path for storing the files retrieved with table get_file.
+    //"getfile_path": "/var/osquery/getfiles",
```

Comment:
Na, no need to add the reference here. Add it to https://github.com/facebook/osquery/blob/master/docs/wiki/installation/cli-flags.md